src/_nested-list.scss

```diff
@@ -0,0 +1,307 @@
+@import "./settings";
+@import "./mixins";
+@import "./functions";
+
+$block: #{$fd-namespace}-nested-list;
+
+@mixin fd-side-nav-arrow {
+  font-family: "SAP-icons";
+  font-style: normal;
+  font-weight: normal;
+  color: var(--sapList_TextColor);
+  font-size: var(--sapFontHeader4Size);
+  text-decoration: inherit;
+  text-transform: none;
+  speak: none;
+}
+
+.#{$block} {
+  @include fd-reset();
+
+  list-style: none;
+  width: 100%;
+
+  @for $i from 2 through 10 {
```

Comment:
I'm not sure if my previous comment got lost or if I forgot to post it, but do we want consuming libraries to have to calculate `.level-#` for classes?

I love that this is a mixin, but it seems like a preferred hierarchy would just be something like this, and the styles would be ready to support all tiers of parent/child relationship (and padding size) up to a certain level.

Thoughts from the rest of the team? (pseudo-code)

```
.list
    .list__item
    .list__item
        .list
            .list__item
            .list__item
       ...
```

Reply:
Brad did rightly mention, though, that the tradeoff for this is making really deep parent-child selectors in the CSS 😕 

`.list > .list__item > list > .list__item {}` etc

I'm not sure which is best

Reply:
@markpalfreeman what would you suggest?

Reply:
We can probably just leave it like this; both ways have tradeoffs.